### PR TITLE
Cron data refresh

### PR DESCRIPTION
**Pull request by parthshahva**
- Cloudwatch event to trigger ecs + fargate data loading every 3 hours
- Store analysis gcp creds in aws secrets manager
- docker image for data refresh

### CODE REVIEW

File: tracker/common/dcp_agents/analysis_agent.py [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-        gcp_credentials_file_for_analysis = os.environ.get('GCP_ACCOUNT_ANALYSIS_INFO')
-        if gcp_credentials_file_for_analysis:
-            with open(gcp_credentials_file_for_analysis, 'r') as f:
-                service_account_key = json.load(f)
-            self.auth = self._get_auth(service_account_key)
+        self.auth = self._get_auth(self.analysis_gcp_creds)
+
+    @property
+    def analysis_gcp_creds(self):
+        return json.loads(base64.b64decode(TrackerInfraConfig().analysis_gcp_creds).decode())
```
Comment (rexwangcc) [inline tracker/common/dcp_agents/analysis_agent.py]: Good change, I guess I need to cherry-pick this to my PR

File: docker-images/data-refresher/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	docker build -t $(IMAGE) .
+	rm -rf code/
+
+release: build bump_version push
+
+push:
+	$(eval VERSION=$(shell cat VERSION))
+	docker tag $(IMAGE):latest $(HUB_ORG)/$(IMAGE):$(VERSION)
+	docker push $(HUB_ORG)/$(IMAGE):$(VERSION)
+
```
[the remark this answers is not in the document]
Comment (parthshahva) [reply docker-images/data-refresher/Makefile]: done

File: docker-images/data-refresher/Makefile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+promote:
+	$(eval VERSION=$(shell cat VERSION))
+	docker tag $(IMAGE):latest $(HUB_ORG)/$(IMAGE):latest
+	docker push $(HUB_ORG)/$(IMAGE):latest
+
+bump_version:
+	expr `cat VERSION` + 1 > VERSION
+
```
[the remark this answers is not in the document]
Comment (parthshahva) [reply docker-images/data-refresher/Makefile]: done

File: terraform/modules/data_refresher_ecs.tf [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+[
+  {
+    "environment": [
+      {
+        "name": "DEPLOYMENT_STAGE",
+        "value": "${var.deployment_stage}"
+      }
+    ],
+    "image": "humancellatlas/data-monitoring-data-refresher:1",
+    "name": "data-monitoring-data-refresher-${var.deployment_stage}",
```
[the remark this answers is not in the document]
Comment (parthshahva) [reply terraform/modules/data_refresher_ecs.tf]: done

**Comment by parthshahva** [comment]
@sampierson @rexwangcc All comments have been addressed. Commits have been rebased into one commit. Merging this in

**Pull request merged**